types/node/index.d.ts

```diff
@@ -21,7 +21,7 @@
 
 /************************************************
 *                                               *
-*               Node.js v8.5.x API              *
+*               Node.js v9.x.x API              *
```

Comment:
@andy-ms already done that

Reply:
The actual header is on the first line -- don't know if this box is really necessary.